pipeline/scripts/7-collectSamplesGroupedHMDB.R

```diff
@@ -93,7 +93,7 @@ n_moved = 0
 
 if(!is.null(outlist_i_min_1)){
   # Calculate 4ppm and replace border, avoid cut within peakgroup!
-  while ((as.numeric(outlist_i[1,"mzmed.pkt"]) - as.numeric(outlist_i_min_1[min_1_last,"mzmed.pkt"]))*1e+06/as.numeric(outlist_i[1,"mzmed.pkt"]) < 2*ppm) {
+  while ((as.numeric(outlist_i[1,"mzmed.pkt"]) - as.numeric(outlist_i_min_1[min_1_last,"mzmed.pkt"]))*1e+06/as.numeric(outlist_i[1,"mzmed.pkt"]) < ppm) {
```

Comment:
Ook hier comment aanpassen. En hier gebeurt dus nu precies hetzelfde als in regel 70, klopt dat?